inference.py

```diff
@@ -0,0 +1,32 @@
+from loader import Preprocessor, StandardConvSuite
+import torch
+
+
+class INFPreprocessor(Preprocessor):

```

Comment:
I don't understand the purpose of this class, doesn't Preprocessor already support setting device?

Reply:
this is necessary due to the fact that in the Preprocessor class in the init function there are actions related to the iterator, which are unnecessary in this case, because there is no iterator